.github/workflows/validate.yml

```diff
@@ -24,29 +24,29 @@ jobs:
         with:
           # See our `rdme` docs for more GitHub Actions usage examples:
           # https://docs.readme.com/docs/rdme
-          rdme: openapi:validate 2.0/json/petstore.json
+          rdme: openapi:validate 2.0/json/petstore.json --workingDirectory=packages/oas-examples
```

Comment:
merging this all in as-is but fyi @kanadgupta in case you have a better way to do this